test/server/lib/DocApi.ts

```diff
@@ -4889,6 +4891,7 @@ function testDocApi() {
 
             const expectedFields = {
               url: `${serving.url}/foo`,
+              authorization: '',
```

Comment:
Maybe also add a specific tests to ensure that a filled authorization header is passed to the webhook consumer?

Reply:
Good remark, adding this test I find a bug when calling patch call : before when we used patch call to update URL without giving information of authorization it removed it